internal/sbi/consumer/nrf_service.go

```diff
@@ -84,59 +85,36 @@ func (s *nnrfService) SendSearchNFInstances(
 		return nil, openapi.ReportError("nrf not found")
 	}
 
-	ctx, _, err := ausf_context.GetSelf().GetTokenCtx(models.ServiceName_NNRF_DISC, models.NfType_NRF)
+	ctx, _, err := ausf_context.GetSelf().GetTokenCtx(models.ServiceName_NNRF_DISC, models.NrfNfManagementNfType_NRF)
 	if err != nil {
 		return nil, err
 	}
 
-	result, res, err := client.NFInstancesStoreApi.SearchNFInstances(ctx, targetNfType, requestNfType, param)
+	res, err := client.NFInstancesStoreApi.SearchNFInstances(ctx, &param)
+	result := res.SearchResult
```

Comment:
@TYuan0816 Please do the error handling first before the assignment. The `res` is probably `nil` if there's an error, which will cause a nil pointer to be dereferenced.

Reply:
Thank you!